src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker

```diff
@@ -27,6 +27,7 @@ kibana_vars=(
     console.proxyFilter
     csp.strict
     csp.warnLegacyBrowsers
+    csp.unsafe_eval
```

Comment:
If you rename the config setting as I suggested above, don't forget to update this accordingly :+1:

Reply:
Done 👍